packages/sui-mono/bin/sui-mono-phoenix.js

```diff
@@ -49,6 +49,10 @@ program
     '--no-progress',
     'Force to not show progress of tasks (perfect for CI environments)'
   )
+  .option(
+    '-s, --scope <string>',
```

Comment:
You could make this an array, so you could spend more than one scope here.

ex.: https://github.com/SUI-Components/sui/blob/master/packages/sui-bundler/bin/sui-bundler-dev.js#L38-L46

Reply:
Lets remind this as a possible future iteration as it's on top of what this task was meant to acomplish